.github/workflows/push_tests.yml

```diff
@@ -41,7 +41,39 @@ jobs:
         run: |
           pip install -r cellxgene_schema_cli/requirements.txt
           pip install pytest
-      - name: Unit tests
+      - name: cellxgene_schema_cli Unit tests
         run: make unit-test
       - name: Upload coverage to Codecov
         uses: codecov/codecov-action@v2
+
+  ontology-dry-run-unit-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install -r schema_bump_dry_run_scripts/ontologies/requirements.txt
+          pip install pytest
+      - name: ontology dry run changes
+        uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            ontology_dry_run:
+              - 'schema_bump_dry_run_scripts/**'
```

Comment:
Since this job only needs to run when these files change, I recommend putting this job in it's own GHA file, that way it only runs when those files change. Otherwise we will run through the whole setup for every PR.